website/docs/cloud-docs/workspaces/settings/notifications.mdx

```diff
@@ -39,12 +39,13 @@ Go to the workspace's notifications settings and press the "Add Notification" bu
 
 Fill in the following fields to configure your notifications:
 
-* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
+* **Destination:** Terraform Cloud can deliver either a generic payload, a payload formatted specifically for Slack, Microsoft Teams, or an Email. Choose which type you want from the drop-down. Payload contents are [documented below][inpage-payload].
 
 * **Name:** An arbitrary display name to use for this notification configuration.
 
-* **Webhook URL** _(only available for generic and slack webhooks):_ The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
+* **Webhook URL** _(only available for generic, Slack, and Microsoft Teams webhooks):_ The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
```

Comment:
I don't think we need the italics --> our new style guidelines say that we should avoid using italics when possible! 

```suggestion
* **Webhook URL** Only available for generic, Slack, and Microsoft Teams webhooks. The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
```
```suggestion
* **Webhook URL** _(only available for generic, Slack, and Microsoft Teams webhooks):_ The destination URL to send the webhook payload to. This URL must accept HTTP or HTTPS `POST` requests, and should be able to do something useful with the chosen payload type.
```